blueprints/posts-via-wp-cli/blueprint.json

```
@@ -0,0 +1,21 @@
+{
+    "$schema": "https://playground.wordpress.net/blueprint-schema.json",
+    "meta": {
+        "title": "Use wp-cli command to add posts",
+        "description": "Blueprint example to add posts via a wp-cli command.",
+        "author": "bph",
+        "categories": ["Content", "wpcli"]
+    },
+    "landingPage": "wp-admin/edit.php",
```

This won't work without the preceeding slash:

```suggestion
    "landingPage": "/wp-admin/edit.php",
```

Now that I think of it, Blueprints could be smart about it and add that slash for you. But until that's the case, let's make sure to explicitly provide one.